### PR TITLE
{ServiceBus} : fix for update methods and split test in individual files

### PR DESCRIPTION
**Pull request by v-Ajnava**
**Description**  

**Testing Guide**  

**History Notes**  

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.  
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [X] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [X] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
add to S169

File: src/azure-cli/azure/cli/command_modules/servicebus/tests/latest/test_servicebus_commands_Test.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
+# --------------------------------------------------------------------------------------------
```
Comment (qwordy) [inline src/azure-cli/azure/cli/command_modules/servicebus/tests/latest/test_servicebus_commands_Test.py:1]: Usually filename is in lowercase.

File: src/azure-cli/azure/cli/command_modules/servicebus/custom.py [hunk 2]
```diff
@@ -436,28 +424,33 @@ def cli_migration_start(client, resource_group_name, namespace_name, target_name
...
+        day, minute, seconds = value_toreturn.split(":")
+        if timedelta(days=int(day), minutes=int(minute), seconds=int(seconds)) < timedelta(days=10675199,
+                                                                                           minutes=10085,
+                                                                                           seconds=477581):
+            return timedelta(days=int(day), minutes=int(minute), seconds=int(seconds))
+
+        if timedelta(days=int(day), minutes=int(minute), seconds=int(seconds)) > timedelta(days=10675198,
+                                                                                           minutes=10085,
+                                                                                           seconds=477581):
+            return timedelta(days=10675198, minutes=10085, seconds=477581)
```
Comment (qwordy) [inline src/azure-cli/azure/cli/command_modules/servicebus/custom.py:452]: Code style can be improved. Avoid magic numbers. Maybe timedelta object can be reused.

**Comment by qwordy** [comment]
Curious about what's the relation with eventgrid

File: src/azure-cli/azure/cli/command_modules/servicebus/tests/latest/test_servicebus_commands_Test.py (hunk 1 above)
Comment (v-Ajnava) [reply src/azure-cli/azure/cli/command_modules/servicebus/tests/latest/test_servicebus_commands_Test.py]: this is used test script, i forgot to delete it before submitting the PR. I have deleted it now

File: src/azure-cli/azure/cli/command_modules/servicebus/custom.py (hunk 2 above)
Comment (v-Ajnava) [reply src/azure-cli/azure/cli/command_modules/servicebus/custom.py:452]: in the update methods, the properties of type timespan are returned with extra day '10675199' and when its passed as it is to the service we get the error as the value is to large. so compared the days.

**Comment by yonzhan** [comment]
add to S170

**Comment by v-Ajnava** [comment]
Hi @fengzhou-msft , @qianwens and @zhoxing-ms , 

Can you please review the PR?

**Comment by yonzhan** [comment]
add to S171

File: src/command_modules/azure-cli-servicebus/azure/cli/command_modules/servicebus/tests/latest/test_servicebus_commands_netwrok.py [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+        self.cmd(
+            'servicebus topic authorization-rule keys list --resource-group {rg} --namespace-name {namespacename} --topic-name {topicname} --name {authoname}')
+
+        # Regeneratekeys - Primary
+        self.cmd(
+            'servicebus topic authorization-rule keys renew --resource-group {rg} --namespace-name {namespacename} --topic-name {topicname} --name {authoname} --key {primary}')
+
+        # Regeneratekeys - Secondary
+        self.cmd(
+            'servicebus topic authorization-rule keys renew --resource-group {rg} --namespace-name {namespacename} --topic-name {topicname} --name {authoname} --key {secondary}')
```
Comment (Juliehzl) [inline src/command_modules/azure-cli-servicebus/azure/cli/command_modules/servicebus/tests/latest/test_servicebus_commands_netwrok.py:90]: Is there any output keyword to check here?

**Comment by Juliehzl** [comment]
@v-Ajnava Can you take a look for test failures?

**Comment by yonzhan** [comment]
@v-Ajnava this PR stays for long time. Close it first. Please let us know if you still want it.

**Pull request closed without merging**